liberty-managed/src/test/java/io/openliberty/arquillian/managed/needsmanagementmbeans/WLPContextRootTestCase.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2010-2012, Red Hat Middleware LLC, and individual contributors
```

Comment:
I assume this has been largely copied from another test? We at least need to update the copyright year.